example/simple_testing/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "simple_testing"
```

Comment:
issue: simple_testing is no longer needed and can be yeeted

Reply:
yeet!